### ISSUE TITLE
Working with Frame parameters for Replica feature fusion

### ISSUE THREAD

**Issue by pratikratadiya** [pengsongyou/openscene#12]
Thank you for this great repository! The documentation is very clear too.

I have a recording of a space with no semantic labels. So I am using the Replica dataset processing files. I was able to preprocess the 3D ply file. For 2D preprocessing, I have all the frames, but it is also asking me for a trajectory.txt file. What is this file supposed to contain? The app I used for getting the data did not provide this file. Instead, for each frame, I get a JSON whose structure is like this:

```
{
  "intrinsics" : [
    1327.18115234375,    0,     959.62261962890625,     0,     1327.18115234375,     720.123046875,
    0,     0,     1
  ],
  "averageAngularVelocity" : 0.83692288398742676,
  "motionQuality" : 0.76087915897369385,
  "frame_index" : 91,
  "exposureDuration" : 0.01666666753590107,
  "averageVelocity" : 0.41746380925178528,
  "cameraPoseARFrame" : [
    -0.35722801089286804,     0.7659410834312439,     -0.53453010320663452,     0.63373279571533203,
    -0.8567434549331665,     -0.040781203657388687,     0.51412779092788696,    0.10553276538848877,
    0.37199276685714722,     0.64161604642868042,     0.67078316211700439,     -1.9972312450408936,
    0,     0,     0,     1
  ],
  "cameraGrain" : 0,
  "time" : 98064.684786291691,
  "projectionMatrix" : [
    1.3824803829193115,     0,     -0.00012767314910888672,     0,     0,     1.8433071374893188,
    0.00086534023284912109,     0,     0,     0,     -0.9999997615814209,     -0.00099999981466680765, 
    0,     0,     -1,     0
  ]
}
```

Which parameter is supposed to be used out of these to get the trajectory.txt? Thanks in advance!

**Comment by pengsongyou** [comment]
Hi @pratikratadiya 

You can use [the script](https://github.com/cvg/nice-slam/blob/master/scripts/download_replica.sh) provided in NICE-SLAM to download the unprocessed Replica data, which contains the traj.txt file. You need to find a way to adjust your .json so that the [2D Replica data processing code](https://github.com/pengsongyou/openscene/blob/main/scripts/preprocess/preprocess_2d_replica.py) is working.

**Comment by pratikratadiya** [comment]
Hi, thanks for responding. I was able to get this running by combining all the "cameraPoseARFrame" attributes. I got the 3D features too, and now when I run the inference script (I am using pretrained model, not distilling my own), I get out of memory error.

I have 4 x 16 GB GPU, but did not modify the code so I think it is only able to use a single GPU 0. How can I modify the config/replica/ours_openseg_pretrained.yaml file to be able to use multiple smaller GPUs for the predictions?
